frontend/app/student/run-simulation/[instanceId]/page.tsx

```
@@ -1596,9 +1679,7 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
 
           {/* Draggable Border */}
           <div
-            className={`w-1 bg-gray-200 hover:bg-gray-300 cursor-col-resize flex-shrink-0 transition-colors ${
-              isDragging ? 'bg-blue-300' : ''
-            }`}
+            className="w-1 bg-gray-200 hover:bg-gray-300 cursor-col-resize flex-shrink-0 transition-colors"
             onMouseDown={handleMouseDown}
           >
             <div className="w-full h-full flex items-center justify-center">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)